rules/list-role-definitions-in-acr/raw.rego

```diff
@@ -0,0 +1,20 @@
+package armo_builtins
+
+# return ListEntitiesForPolicies resource in azure
+deny[msg] {
```

Comment:
if this is an "empty" rule, we can consider to use, as best practice, the `rule-manual` rule that I'm implementing here: https://github.com/kubescape/regolibrary/blob/a7c6aa2968965b2f9d6651fe4d9f705466240afb/rules/rule-manual/raw.rego

Reply:
This rule returns the ListEntitiesForPolicies resource so not completly empty, it's similar to control 5.1.2 of cis-eks that you wrote :)